frontend/app/src/types/history/calendar/index.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from 'zod';
+import { Blockchain } from '@rotki/common/lib/blockchain';
+import { CollectionCommonFields } from '@/types/collection';
+import type { PaginationRequestPayload } from '@/types/common';
+
+const BlockchainEnum = z.nativeEnum(Blockchain);
+
+export const CalendarEventPayload = z.object({
+  name: z.string(),
+  description: z.string(),
+  counterparty: z.string(),
+  timestamp: z.number(),
+  address: z.string().optional(),
+  blockchain: BlockchainEnum.optional(),
+  color: z.string().nullable(),
```

Comment:
shouldn't this also be optional instead of nullable?

The api documentation says optional, @yabirgb is there any chance that the backend will return `color: null`?

Reply:
yeah it's null rn, my bad, I asked yabir to make it nul

Reply:
Is there any reason we want it to be null instead of missing when it is not set?

Reply:
no, no reason, this should be optional instead. I will update it